docs/java-helloworld.md

```diff
@@ -0,0 +1,214 @@
+
+<img class="docs-image-centered" src={require('../static/img/docs/hello.png').default} />
+
+This tutorial focuses on the practicalities of building an application from scratch. To better understand *why* you should use Temporal, we recommend that you follow the tutorial where you [run a Temporal money transfer application](/docs/java-run-your-first-app) to get a taste of its value propositions.
+
+All of the code in this tutorial is available in the [hello-world Java template repository](https://github.com/temporalio/hello-world-project-template-java).
+
+## ![](/img/docs/harbor-crane.png) &nbsp;&nbsp; Scaffold Gradle
+
+Before starting, make sure you have looked over the [tutorial prerequisites](/docs/java-sdk-tutorial-prerequisites).
+
+Create a new project directory called "hello-world-tutorial", or something similar.
+
+You can scaffold a new Gradle project from the terminal or from within IntelliJ.
+
+**Terminal:**
+
+Change your working directory to the one created for the project and follow Gradle's [Building Java Applications](https://guides.gradle.org/building-java-applications/) guide. When you get to the step where you define your source package, use "helloworldapp".
+
+**IntelliJ**
+
+Open IntelliJ and create a new Gradle project by following Step 1 of the [Getting started with Gradle guide](https://www.jetbrains.com/help/idea/getting-started-with-gradle.html#create_project). When you get to the step where you name the project, use
+"helloworldapp" and make sure you choose the "hello-world-tutorial" directory as the project location. It will take a few moments to complete.
+
+## ![](/img/docs/block.png) &nbsp;&nbsp; Project dependencies
+
+Once Gradle has finished scaffolding you will need to customize the project dependencies. To do this, open the build.gradle file that is in the root of your project and add the following lines to the dependencies section. If you want to try using different versions of dependencies, you can find them on [search.maven.org](https://search.maven.org/) ([Temporal SDK versions](https://search.maven.org/artifact/io.temporal/temporal-sdk)):
+
+<!--SNIPSTART hello-world-project-template-java-gradle-dependencies-->
+<!--SNIPEND-->
+
+- `com.google.guava:guava` offers a suit of core and expanded libraries that Gradle uses.
+- `io.temporal:temporal-sdk` enables communication with the Temporal server.
+- `ch.qos.logback:logback-classic` will ensure that there is a logger to bind to within the SDK and prevent a default logger warning message.
+
+A "refresh" icon will appear on the screen, click it to load the changes. Gradle will rebuild with the dependencies.
+
+To limit the logging output from the SDK, within src/main/resources/ create a logback.xml file and paste in the following XML:
+
+<!--SNIPSTART hello-world-project-template-java-logback-dependency-configuration-->
+<!--SNIPEND-->
```

Comment:
(just asking, maybe the answer is no) a lot of setup. is there any way to collapse this into a single command?

Reply:
Unfortunately no -
But that could be a cool tool to create - Temporal Java project scaffold tool
But at present these are the steps to create a project from scratch - which is the intent of the tutorial.